data_taking_scripts/dripline_logger.py

```diff
@@ -0,0 +1,29 @@
+from dripline.core import Interface
+import time
+class DriplineLogger:
+
+    def __init__(self, auths_file):
+        self.auths_file = auths_file
+        self.cmd_interface = Interface(dripline_config={'auth-file': self.auths_file})
+
+    def log_modemap(self, sleep_time):
```

Comment:
@parashar2399 this function is not named properly. This does not take the entire modemap measurement. It just takes a logs a single s11 and a single s21.